packages/react-native-bpk-component-navigation-bar/src/TitleView.js

```diff
@@ -45,11 +45,13 @@ const styles = StyleSheet.create({
 
 const TitleView = (props: {
   title: string | TitleWithIcon,
-  tintColor: string,
+  tintColor: ?string,
   style: any,
 }) => {
   const { title, tintColor, style, ...rest } = props;
 
+  const tintColorFinal = tintColor || colorGray700;
+
```

Comment:
This change means that internal styling will be applied as a fallback if `tintColor` is not passed in, which is the same as how the iOS text button works. This is necessary because in `BpkNavigationBar.ios.js`, `tintColor` now defaults to `null`.

As this file applies to Android too, technically I've put an iOS style in here (`colorGray700`). In practice this isn't an issue because `BpkNavigationBar.android.js` *always* passes in a white `tintColor`, but I can tweak this file to do a platform check and make the default Android colour white, if we prefer to make it safer.